functions/go/search-replace/README.md

```diff
@@ -142,6 +154,36 @@ a:
     f: thingamabob
 ```
 
+### File path patterns
+
+`--by-file-path` matcher supports the following special terms in the patterns:
+
+| Special Terms | Meaning                                                                                                   |
+| ------------- | --------------------------------------------------------------------------------------------------------- |
+| `*`           | matches any sequence of non-path-separators                                                               |
+| `/**/`        | matches zero or more directories                                                                          |
+| `?`           | matches any single non-path-separator character                                                           |
+| `[class]`     | matches any single non-path-separator character against a class of characters ([see "character classes"]) |
+| `{alt1,...}`  | matches a sequence of characters if one of the comma-separated alternatives matches                       |
+
+Any character with a special meaning can be escaped with a backslash (`\`).
+
+A doublestar (`**`) should appear surrounded by path separators such as `/**/`.
```

Comment:
this contradicts the setters example below: `by-file-path='**/setters.yaml'`

Reply:
Yes. Removed this line.